### PR TITLE
yea there's a slight chance I got carried away... :D

### PR DESCRIPTION
**Pull request by Amheus**
- now uses slash commands
- timing system hopefully a little more consistent
- now stores targeted ChannelID in the `.env` file
- only drank three cups of tea during all of this
- now stores video links as a Dict ~~because Python Dicts are cool~~
- definitely didn't get carried away with the README.md file

### CODE REVIEW

File: fixed README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 <a href="#3. Requirements">3. Requirements</a><br>
 <a href="#3.1. Packages">3.1. Packages</a><br>
 <a href="#3.2. .env File">3.2. .env File</a><br>
-<a href="#3.2.1 DISCORD_TOKEN">3.2.1 `DISCORD_TOKEN`</a><br>
-<a href="#3.2.2 CHANNEL_ID">3.2.2 `CHANNEL_ID`</a><br>
+<a href="#3.2.1. DISCORD_TOKEN">3.2.1 `DISCORD_TOKEN`</a><br>
+<a href="#3.2.2. CHANNEL_ID">3.2.2 `CHANNEL_ID`</a><br>
```
Comment (JoelLucaAdams) [inline fixed README.md]: no sure what the hell happened here with all these HTML tags... Doesn't render properly

File: .gitignore [hunk 2]
```diff
@@ -1,7 +1,3 @@
+
 .env
 .idea/
-__pycache__/
-Pipfile.lock
-discord.log
-stressLevels.json
-.vscode/
```
Comment (JoelLucaAdams) [inline .gitignore:7]: Please re-add the .vscode/ folder as it is used when editing the file in VS code

File: requirements [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+discord
+discord.py
+discord-py-slash-command
+python-dotenv
```
Comment (JoelLucaAdams) [inline requirements:5]: I understand why you added this but then why did you delete the pipfile? pipfiles are a lot more usable than a requirements.txt

File: fixed README.md [hunk 4]
```diff
@@ -31,20 +31,22 @@ This will automatically download and install the packages listed in the `require
...
-
-<h3>3.2.2. `CHANNEL_ID`</h2>
+ <ol>
+  <li>Going to https://discord.com/developers/applications`.</li>
+  <li>Clicking on the bot.  If you have not yet created a bot, then click `New Application` in the top right.</li>
+  <li>Navigate to the `Bot` menu on the left-hand side.</li>
+  <li>Click `Copy` under `Click to Reveal Token` and paste into the `.env` file.  If you haven't created a bot, then press `Add Bot`, then `Yes, do it!`.</li>
+</ol>
+
+<h3 id="3.2.2. CHANNEL_ID">3.2.2. `CHANNEL_ID`</h2>
```
Comment (JoelLucaAdams) [inline fixed README.md]: Really not a fan of using HTML so much in the readme.md file as it breaks the standards and really isn't needed over a plain markdown file

**Review by JoelLucaAdams — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (Amheus) [reply .gitignore:7]: alrighty, will do :)

File: fixed README.md (hunk 4 above)
Comment (Amheus) [reply fixed README.md]: I'm using HTML in the README file for the contents page so you can jump to the section headers.
Probably isn't needed but I threw it in for uh... reasons... definately not because I was bored...

File: requirements (hunk 3 above)
Comment (Amheus) [reply requirements:5]: Now that one *wasn't* intentional, I kept it in the project because there wasn't any need to remove it then before committing I was doing some tidying up, saw a file I wasn't using and removed it, forgetting that I didn't make that file lol.

**Review by JoelLucaAdams — APPROVED** [review]

**Comment by JoelLucaAdams** [comment]
Thanks for the updated changes :D Will merge now
